cocos/core/scheduler.ts

```diff
@@ -35,19 +35,27 @@ const MAX_POOL_SIZE = 20;
 const idGenerator = new IDGenerator('Scheduler');
 
 export interface ISchedulable {
-    id?:string;
-    uuid?:string;
+    id?: string;
+    uuid?: string;
 }
 
 // data structures
 /**
  * @en A list double-linked list used for "updates with priority"
  * @zh 用于“优先更新”的列表
  * @class ListEntry
- * @param target not retained (retained by hashUpdateEntry)
+ * @param target
+ * @en target object, which is ISchedulable type, retained by hashUpdateEntry
+ * @zh 目标对象, 为ISchedulable类型. 被hashUpdateEntry持有

```

Comment:
- Need `.` or `。` at the end of a sentence
- Should use capital character for at the begin of a sentence, so should change `target` to `Target`.

Reply:
Got it. Done.